contents/handbook/people/compensation.md

```diff
@@ -88,3 +88,13 @@ Unless there is a very good reason, you'll be paid in your local currency. This
 | GBP | 0.758 |
 
 </span>
+
+## Probation
+
+You will join us on a 3 month probation period, during which either you or PostHog can choose to end your contract with 1 week's notice. We are fully committed to ensuring that you are set up for success, but also understand that it may take some time to determine whether or not there is a long term fit between you and PostHog. Our bar for hiring is very high, and you should not expect to automatically pass probation. 
+
+Your manager is responsible for monitoring and specifically reviewing your performance throughout the probation period. If underperformance is a concern, or if there is any hesitation regarding the successful completion of a probation period, this should be discussed immediately with you and James or Tim. 
+
+## Severance
+
+If your time at PostHog comes to an end, then we will pay generous severance. If PostHog decides to end your contract after your probation period has been completed, we will pay you 4 months of salary. If the decision to leave is yours, then we just require 1 month of notice. We have structured notice in this way as we believe it is in neither PostHog's nor your interest to lock you into a role that is no longer right for you due to financial considerations. This extended notice period only applies in the case of under-performance or a change in business needs - if your contract is terminated due to gross misconduct then you may be dismissed without notice.
```

Comment:
A couple of thoughts:
- Generous severance is relative, I think there are some countries where the minimum would be 6 months (AR comes to mind). I believe Netflix's approach is generous severance relative to the circumstances (unsure).
- I also think worth mentioning that all of this is in addition (or does not substitute) any requirements by law.
- It is not clear if the 1-month notice period applies to PostHog's decision of termination or just the employee's.
- There's some countries where even voluntary termination by the employee requires some type of severance pay (not really severance, but there's some sort of payment here), which maybe should be reflected too?

Reply:
Thanks @paolodamico ! My comments below:

- Yes you are quite right - I copied the use of 'generous' from another page, but agree I approached this with a US & Europe bias. Typically severance is tied to years of service (e.g. 1 week for every year you've worked), so I would class today's severance of 4 months as generous given no one has worked at PostHog for more than a few months, but this may not be the case in the future. Let me try and rephrase and see what you think. 
- Agreed, I will add this. 
- Good point, I will make this clearer. 
- This should be captured by your second point for those countries when they arise.

Reply:
A few follow-ups (obs feel free to disregard),
- From reading the paragraph it's still ambiguous to me if the "generous severance" is 4 months of pay on the last day, or if the generous severance is understood from giving 4 months notice.
- Personally I would remove the notice period for the company, realistically it could be 2 weeks or 6 months (think it'll be a case-by-case basis), and ultimately the most flexibility is on not having a set policy.

LGTM otherwise!

Reply:
I tweaked the text to address your first bullet, so hopefully that is now clearer!

We need to leave the notice period in - if we don't define one, then it will just default to whatever is in someone's contract (where we have to put in a notice period). So it's more sensible to give a long one by default but give flexibility to revise this downwards if it makes sense.

Reply:
That makes sense, and thanks for clarifying! Sorry for the push back (and feel free to discard) but I would argue if we give 4 months notice to someone and then just pay for those 4 months we're not really giving a generous severance (or any severance at all). To my understanding severance is a payment after employment has been terminated, but this might just be a misconception on my part.

Reply:
I agree with you, and I guess it 'becomes' severance pay if we agree a shorter notice period than 4 months (if that makes sense...)

Should we just keep it simple and shorten the company notice period to 1 month then as well? ie. company gives 1 month's notice, you get paid for 4 months, and we may shorten this further in agreement with the employee? 

I think I may be overcomplicating a relatively straightforward issue here.

Reply:
Hm I'm not sure where this confusion has come from. I think in the actual diff it's explained well. If we decide to end your contract, we'll give you 4 months pay, you'll stop working immediately (or very shortly). If you decide to leave we require one months notice.

The only thing that might be worth clarifying is the fact that if we decide to terminate you will likely stop working immediately.

Reply:
Yup think that was the missing piece, if you stop working immediately that makes total sense, sorry for the back and forth here.

Reply:
Just added that - let's push? I think we've finessed this as much as we need to now!

Reply:
agreed 👍